shared/models.go

```diff
@@ -69,6 +70,32 @@ type WebSocketTapConfigMessage struct {
 	TapTargets []v1.Pod `json:"pods"`
 }
 
+type ServiceMapStatus struct {
```

Comment:
i think we should start splitting this to separate files, so this won't become a huge file with all application models..
can you create a models file just for servicemap structs?

Reply:
sure, this is a good idea, but each package will have its own models file? or we will have new models file inside the "shared" package?